packages/source-iottwinmaker/src/types.ts

```diff
@@ -1,10 +1,18 @@
+import { GetSceneCommandOutput } from '@aws-sdk/client-iottwinmaker';
+import { SecretListEntry } from '@aws-sdk/client-secrets-manager';
 import { VideoPlaybackMode } from './video-data/types';
 
 export interface SceneLoader {
   getSceneUri: () => Promise<string | null>;
   getSceneObject: (uri: string) => Promise<ArrayBuffer> | null;
 }
 
+export interface TwinMakerSceneMetadataModule {
+  getSceneInfo: () => Promise<GetSceneCommandOutput>;
+  updateScene: (capabilities?: string[], sceneMetadata?: Record<string, string>) => Promise<void>;
```

Comment:
maybe `updateSceneInfo`? so it's consistent with `getSceneInfo` and also won't have confusion with whether it updates the scene file in s3 or the twinmaker scene object

Reply:
also it should take just one parameter with type 
```
{
  capabilities?: string[], 
  sceneMetadata?: Record<string, string>
}
```
so if update scene support more parameters, we will just add more fields to this type, instead of add more parameters to the function. 

potentially this type should be the type of the input for `UpdateSceneCommand` but omitting workspaceId and sceneId  (see https://stackoverflow.com/questions/48215950/exclude-property-from-type)

Reply:
Renamed to updateSceneInfo() with named parameter